src/Build/Logging/BaseConsoleLogger.cs

```diff
@@ -332,9 +265,9 @@ internal void ShowPerfSummary()
         /// </summary>
         internal void WritePretty(int indentLevel, string formattedString)
         {
-            StringBuilder result = new StringBuilder();
```

Comment:
Is it really worth the reservation? Shouldn't this come down to two strings: a bunch of spaces and `formattedString`, and then a single concatenation?